packages/generator/templates/mutations/delete__ModelName__.ts

```diff
@@ -1,13 +1,9 @@
 import { resolver } from "@blitzjs/rpc"
 import db from "__prismaFolder__"
-import {z} from "zod"
-
-const Delete__ModelName__ = z.object({
-  id: z.__modelIdZodType__(),
-})
+import {Delete__ModelName__Schema} from "../validations/schema"
```

Comment:
I prefer keeping the the file specific schema in the same file, especially since it is very small. I never needed to export and use it somewhere else.

Reply:
Yeah, I also thought about this, but I added this to maintain consistency in the way the schema is handled in the mutations folder.